CIP-0071/README.MD

```diff
@@ -0,0 +1,238 @@
+---
+CIP: 0071
+Title: Verified Pool Identity
```

Comment:
```suggestion
Title: Verified Stake-Pool Identity
```

Reply:
don't want to nitpick after my last review comment, only to address comments so far: there is no hyphen in "stake pool" (reference link below)